services/scabbard/libscabbard/src/store/mod.rs

```diff
@@ -19,7 +19,7 @@ mod command;
 mod commit_hash;
 #[cfg(any(feature = "postgres", feature = "sqlite"))]
 pub(crate) mod pool;
-#[cfg(feature = "scabbardv3")]
+#[cfg(feature = "scabbardv3-store")]
```

Comment:
This feature should also guard the pub use statements below

Reply:
Fixed